openlibrary/templates/books/works-show.html

```diff
@@ -3,7 +3,7 @@
 <span itemscope itemtype="https://schema.org/Book">
   <span class="bookcover">
     $ url = book.get_cover_url("S") or "/images/icons/avatar_book-sm.png"
-    <a href="$book.key"><img itemprop="image" src="$url" height="60"/></a>
+    <a href="$book.key"><img itemprop="image" src="$url" srcset="$url" height="60"/></a>
```

Comment:
@devarshigoswami the srcset attribute should point to a higher resolution url
Note that both of these attributes are pointing to a small version of the cover using `book.get_cover_url("S")` - I suggest you look at the get_cover_url method and request a medium cover size using the relevant parameter for the srcset attribute (src value is fine).

The srcset syntax is also a little different - check out [the spec](https://html.spec.whatwg.org/multipage/images.html#srcset-attribute) and [some examples](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). You'll likely want to use the 2x value to fix this particular issue.

Reply:
@jdlrobson 
I understood what `get_cover_url(item_id)` is doing . Is there a medium-sized image for item_id = "M" in the location that function is pointing to ?
If there is, from what I perceive, we can just create a variable for a medium-sized image coverMedium = book.get_cover_url('M') which I can point to `srcset="$urlB "`?? 
2)Can the pixel density be set as ` $ urlB =book.get_cover_url("M")  2x` ?
3)If that does , just will `srcset= " $url ,  $urlB 2x "` suffice for all possible resolutions?


**I have a few questions regarding another function I found interesting:**
`if coverLarge:
 self.create_rel_link(None, 'http://opds-spec.org/image', coverLarge, 'image/jpeg')` in plugins/openlibary/opds.py which has the `create_rel-link` method which has a parameter  `http://opds-spec.org/image`
what does create_rel_link do and what is the opds URL?